.github/workflows/run_benchmarks.yml

```diff
@@ -0,0 +1,61 @@
+name: Run Benchmarks
+
+on:
+  pull_request:
+    branches:
+      - master
+      - 'releases/**'
+
+jobs:
+  benchmark:
+    container:
+      image: vowpalwabbit/rlclientlib-manylinux2010-build:latest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          ref: ${{ github.base_ref }}
+      - name: Install google benchmarks
+        shell: bash
+        run: ./ext_libs/vowpal_wabbit/.scripts/linux/install-benchmarks.sh
+      - name: Build ${{ github.base_ref }}
+        shell: bash
+        run: ./.scripts/linux/build-with-benchmarks.sh
+      - name: Benchmark ${{ github.base_ref }}
+        shell: bash
+        run: ./.scripts/linux/run-benchmarks.sh master-benchmarks.json
+      - name: Upload ${{ github.base_ref }} benchmark results
+        uses: actions/upload-artifact@v2
+        with:
+          name: master-benchmarks
+          path: master-benchmarks.json
+      - name: Upload benchmark compare
+        uses: actions/upload-artifact@v2
+        with:
+          name: benchmark-compare
+          path: benchmark/tools/
+      - name: Install benchmark compare requirements
+        shell: bash
+        run: /usr/local/miniconda/envs/test-python36/bin/pip install -r benchmark/tools/requirements.txt
+      - run: rm -rf benchmark build rlclientlib/generated/ # generated or downloaded files
+      - uses: actions/checkout@v1
+        # temp for first build
```

Comment:
lines 43 and 44 will run the branch against the branch since the benchamarks do not exist in master, should be removed before merging so that future branches run against master